src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -310,6 +310,9 @@ public SubstrateDispatcher(Path buildRoot, ProjectConfiguration config) throws I
     }
 
     private TargetConfiguration getTargetConfiguration(Triplet targetTriplet) throws IOException {
+        if (!config.getHostTriplet().canCompileTo(targetTriplet)) {
```

Comment:
I don't think this check belongs here. We want to get the targetConfiguration, even if we can't compile to it.
(e.g. because we want to build in the cloud, or do other operations, e.g. link/run or so, that do not depend on compilation)

Reply:
Ok, but problem so far is that AndroidTarget will start downloading SDK once the target is set.

Reply:
That is wrong, but has nothing to do with cross-compiling.
e.g. farm build allows for AndroidTarget on mac, but SDK should not be downloaded.